README.md

```diff
@@ -131,10 +131,16 @@ $ knife vsphere vm clone MACHINENAME --template TEMPLATENAME --bootstrap --cips
   --cspec SPEC --ssh-user USER --ssh-password PASSWORD
 ```
 
-- Deleting a machine.
+Note: add a `-f FOLDERNAME` if you put your `--template` in someplace other then
+root folder, and the `--dest-folder FOLDERNAME` to the folder you want to put it in.
```

Comment:
I know `it` is referring to the VM we're creating, but I'm not sure it would be clear to someone new to the project. Could I suggest `and use --dest-folder FOLDERNAME if you want your VM created in FOLDERNAME rather than the root.`